Scripts/Morphology_main.py

```diff
@@ -6,49 +6,40 @@
 @author: thibault
 """
 import Traits_class as tc
-import json, sys, math
+import json
 import numpy as np
+import argparse
 
 def get_scale(metadata_file):
-
-    '''
-    extract the scale value from metadata file
     '''
-    
-    f = open(metadata_file)
-    data = json.load(f)
-    metadata_dict = list(data.values())[0]
-    scale = 'None'
-    unit = 'None'
-
-    if 'scale' in metadata_dict  :
-        scale = round(metadata_dict['scale'],3)
-        unit = metadata_dict['unit']
-        
-    return scale , unit
+    Extract the scale value from metadata file    
 
+    Parameters
+    ----------
+    metadata_file : string
+        DESCRIPTION. .json file containing scale key such as{....{ruler:{scale:xxx, unit:yyy}}}
 
-def get_angle(metadata_file):
+    Returns
+    -------
+    scale : float
+        DESCRIPTION. scale conversion pixel/unit
+    unit : string
+        DESCRIPTION. unit value expected cm or in(inch)
 
     '''
-    Calculate fish orientation from metadata file using major axis
-    return value in degree
     
-    '''
-    f = open(metadata_file)
-    data = json.load(f)
-    metadata_fish = list(data.values())[0]['fish'][0]
-    fish_angle = None
-    major = []
-    length = []
     
-    if 'primary_axis' in metadata_fish  :
+    f = open(metadata_file)
+    metadata_dict = json.load(f)
```

Comment:
Could we wrap this in a `with` clause to automatically close the file? I think this would leak a file handle.

Reply:
good point. Sometime I do it sometime I don't... Thanks for the suggestion, make the code look better and more robust. It is done here
https://github.com/hdr-bgnn/Morphology-analysis/commit/0124795550ffdf5173e879338f24265ff3be6b64